autolab/Autolab
# Add LTI Configuration to "Manage Autolab" Dropdown, Update docs and gitignore

## Description


This PR aims to fill in the documentation gap for LTI Integration. #1769 reworked how LTI configuration settings are implemented on Autolab, and made certain setup instructions no longer relevant. It also required new documentation for Administrators to properly understand how to setup LTI Advantage Integration. There also did not exist any instructions for instructors to use the course linking feature provided by LTI integration as well.

#1709 also replaced the admin page with a dropdown, and so the LTI configuration settings page (/lti_config/index) did not have a link anywhere on the site. This PR addresses this issue by adding a link to the LTI configuration settings page in the new dropdown.

- Created `docs/features/lti-course-integration.md` as a guide for instructors on how to use the LTI course linking feature
- Added link to LTI config page in Manage Autolab dropdown in `_navbar.html.erb`
- Rework `docs/installation/lti_integration.md` to reflect new configuration scheme
- Update OSX and Ubuntu installation guides to remove instructions to copy `lti_settings.yml.template` which no longer exists
- Update `.gitignore` to no longer ignore `lti_settings.yml`, added `lti_config.yml`, `lti_tool_jwk.json` and `lti_platform_jwk.json` (which are the configuration files added in #1769)


## Motivation and Context


- Addresses missing documentation from #1769 

## How Has This Been Tested?




1. Run the documentation server locally
2. View `http://localhost:8000/installation/ubuntu/`, see that instruction to copy `lti_settings.yml.template` no longer exists, and there is a link to LTI Integration setup at the bottom of the instructions
3. View `http://localhost:8000/installation/osx/`, see that instruction to copy `lti_settings.yml.template` no longer exists, and there is a link to LTI Integration setup at the bottom of the instructions
4. View `http://localhost:8000/installation/lti_integration/`, see it reflects new config scheme
5. View `http://localhost:8000/features/lti-course-integration/`, see it describes LTI linking 

## Checklist:
